openwisp_notifications/tests/test_api.py

```diff
@@ -544,19 +546,20 @@ def test_notification_setting_list_api(self):
                     next_response.data['next'],
                 )
             else:
-                self.assertIsNone(next_response.data['next'])
+                self.assertIsNotNone(next_response.data['next'])
 
         with self.subTest('Test individual result object'):
             response = self.client.get(url)
             self.assertEqual(response.status_code, 200)
             notification_setting = response.data['results'][0]
             self.assertIn('id', notification_setting)
-            self.assertIsNone(notification_setting['web'])
-            self.assertIsNone(notification_setting['email'])
+            self.assertTrue(notification_setting['web'])
+            self.assertTrue(notification_setting['email'])
             self.assertIn('organization', notification_setting)
 
     def test_list_notification_setting_filtering(self):
         url = self._get_path('notification_setting_list')
+        tester = self._create_user()
```

Comment:
Can you instead create an organization admin here? It will make the test more realistic.